packages/suite/src/types/trading/tradingForm.ts

```diff
@@ -268,6 +269,7 @@ export interface TradingExchangeFormContextProps
     setExchangeStep: (step: TradingExchangeStepType) => void;
     confirmTrade: (address: string, extraField?: string, trade?: ExchangeTrade) => Promise<boolean>;
     sendTransaction: () => void;
+    signDataAndConfirm: () => void;
```

Comment:
Let's return Promise<void>

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/17005/commits/18d74b1db384e3076aeaeae1518ceeab53e5f89a